### PR TITLE
Added image and input component

### PR DESCRIPTION
**Pull request by rahul31agrawal**
Hi Guys,
I have added image and input component to my component library, please review.

Thanks

### CODE REVIEW

File: cssFiles/image.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+.imageContainer{
+    min-height: 100px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: .5rem;
+    margin: auto;
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
```
Comment (hiteshverma27) [inline cssFiles/image.css:9]: Try to use hex codes and set a variable for shadow if you are using same box shadow everywhere.

File: cssFiles/image.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
+.imageContainer{
```
Comment (hiteshverma27) [inline cssFiles/image.css:1]: Suggestion- Try to use a standard example box to avoid code repetition.

File: cssFiles/input.css [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    
+  }
+  
+  #login-btn:hover{
+    background-color: darkblue;
+  }
+  
+  #create-btn{
+    width: 30%;
+    background-color: white;
```
Comment (hiteshverma27) [inline cssFiles/input.css:49]: suggestion - set colors in variable.css file and use var here

File: cssFiles/input.css [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+  
+  }
+  
+  #check{
+    color: green;
+  }
+  .wrong{
+    color: red;
+  }
+  #password-outer{
```
Comment (hiteshverma27) [inline cssFiles/input.css:74]: Avoid use of ID and use class as much as possible to improve code reusablity.

File: index.html [hunk 5]
```diff
@@ -414,11 +414,162 @@ <h6> Category: Men's t-shirt</h6>
...
+                          <small class = "wrong">Password must have atleast 1 alphaneumeric & 1 special character!</small>
+                        </div>  
+
+                </div>
+
+            </div>
+          </div>  
+         
+          
+    </div>
```
Comment (hiteshverma27) [inline index.html:556]: Use a code formatter to remove these gaps in code and make it cleaner.

**Review by hiteshverma27 — APPROVED** [review]
need minor changes as mentioned in comments, rest looks good.

File: cssFiles/input.css (hunk 4 above)
Comment (Chirag-kumar-dev) [reply cssFiles/input.css:74]: yes u can avoid much use of id,good to go as of now

File: index.html (hunk 5 above)
Comment (Chirag-kumar-dev) [reply index.html:556]: Yes in real world projects ,it is mandatory to avoid extra spaces.

**Review by Chirag-kumar-dev — APPROVED** [review]

File: cssFiles/image.css (hunk 1 above)
Comment (rahul31agrawal) [reply cssFiles/image.css:9]: ok

File: cssFiles/image.css (hunk 2 above)
Comment (rahul31agrawal) [reply cssFiles/image.css:1]: ok

File: cssFiles/input.css (hunk 3 above)
Comment (rahul31agrawal) [reply cssFiles/input.css:49]: ok

File: cssFiles/input.css (hunk 4 above)
Comment (rahul31agrawal) [reply cssFiles/input.css:74]: ok

**Pull request merged**